test-case-reporting/test/fixtures/sample-karma-report.json

```diff
@@ -59,8 +59,8 @@
                 },
                 {
                     "id": "",
-                    "description": "when test was skipped / it skipped",
-                    "suite": [],
+                    "description": "it skipped",
+                    "suite": ["when test was skipped"],
```

Comment:
Let's add one more test case sample here with a multi-part suite (or modify an existing one) so we hit that case for generating the full name

Reply:
Done.